packages/discord.js/src/structures/MessagePayload.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 const { Buffer } = require('node:buffer');
+const { ActionRowBuilder } = require('@discordjs/builders');
```

Comment:
This should be removed completely, since we don‘t want to make new builders in resolveBody() anymore and just keep json as is.